tests/integration/suite/placement/table/vnodes_api_level_10.go

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Please refrain from using underscores as this is not idiomatic to Go, and instead prefer using packages/directories which also helps with the test name somewhat.

We can rename this package & directory from `table` to `vnodes`. We can then rename these test names to `level10`, `level20`, and `upgrade` respectively.